tensorflow/compiler/xla/service/hlo_instructions.h

```diff
@@ -1562,6 +1568,8 @@ class HloCustomCallInstruction : public HloInstruction {
   std::vector<std::pair<ShapeIndex, std::pair<int64, ShapeIndex>>>
       output_to_operand_aliasing_;
   absl::optional<Literal> literal_;
+  // A custom-call schedule hint.
+  CustomCallSchedule custom_call_schedule_;
```

Comment:
You could just add a default initializer (to `CustomCallSchedule::NONE`) here instead of doing it in every constructor.